src/components/button/button.tsx

```diff
@@ -29,9 +30,10 @@ export const Button: FC<IButtonProps> = ({
         `size-${size}`,
         `${fullWidth ? "button-fullwidth" : ""}`,
         `${isLoading ? "button-loading" : ""}`,
+        className,
       ].join(" ")}
     >
-      <p role="label">{children}</p>
+      <p>{children}</p>
```

Comment:
🦆 let's check if this `p` tag is necessary here

Reply:
🥔 I found a way to remove the `p` tag and have edited the style accordingly! 😎